languages/python/setup.cfg

```diff
@@ -13,8 +12,7 @@ disallow_untyped_defs = True
 disallow_incomplete_defs = True
 ignore_missing_imports = True
 
-[isort]
-profile = black
+
 
 [run]
 omit = tests/*
```

Comment:
This file should be removed, right? Together with the pylintrc

Reply:
Yup